internal/flowrate/xrate/monitor.go

```diff
@@ -0,0 +1,98 @@
+package xrate
+
+import (
+	"time"
+
+	"golang.org/x/time/rate"
+
+	legacyimpl "github.com/cometbft/cometbft/internal/flowrate/legacy"
```

Comment:
```suggestion
	"github.com/cometbft/cometbft/internal/flowrate"
```

Reply:
This does not work. The `legacy` contains a package with `flowrate` name.